dashboard/src/components/Header.tsx

```diff
@@ -0,0 +1,102 @@
+import styled from "@emotion/styled";
+import { Button, Sheet, Typography } from "@mui/joy";
+import { FC, useEffect, useRef, useState } from "react";
+import { Link } from "react-router-dom";
+import { ColorSchemeToggle } from "./ColorSchemeToggle";
+
+import soundEffect from "../assets/slap.mp3";
+
+const rainbowGradient = `linear-gradient(
+  to right,
+  rgba(255, 0, 0, 0.7),
+  rgba(255, 165, 0, 0.7),
+  rgba(255, 255, 0, 0.7),
+  rgba(0, 128, 0, 0.7),
+  rgba(0, 0, 255, 0.7),
+  rgba(128, 0, 128, 0.7)
+)`;
+
+const AnimatedSheet = styled(Sheet, { label: "styledComponent" })<{
+  isblue: boolean;
```

Comment:
When adding props, remember to add `shouldForwardProps` option so that these props are not sent on to the html where it will produce errors or warnings

Reply:
Futhermore, I suggest using scss modules for styling for the future. We've had a style convention talk where this is the future direction we'll probably go with.